use-cases/loyalty.md

```diff
@@ -0,0 +1,85 @@
+# Customer loyalty
+
+Loyalty scheme providers can use the **Mews Connector API** to synchronize their loyalty programs with **Mews Operations**.
+Hotel staff can then easily access a guest's membership status, including loyalty tier such as "Gold member", through the guest profile within the Mews system.
+
+> ### Loyalty data model
+>
+> The data model for Loyalty consists of three main entities:
+> * **Loyalty Program**<br>This is the main loyalty scheme or program. A guest can be a member of multiple programs.
+> * **Loyalty Tier**<br>You can optionally define tiers or levels of membership for your program.
+> * **Loyalty Membership**<br>A Loyalty Membership describes the relationship between a customer or guest and a loyalty program. This includes reward points, loyalty tier and membership expiration date.
+
+## Chains
+
+Loyalty programs are set up in Mews at the above-property chain level. However, this does not prevent individual properties (called _Enterprises_ in Mews) from having loyalty programs, because a property by default is in a chain of one.
+
+> ### Where can I obtain the Chain identifier?
+>
+> You can obtain the Chain identifier for an enterprise or property using [Get configuration](../operations/configuration.md#get-configuration).
```

Comment:
Sorry in advance but I'm going to complicate things for you. Do you really want to mention stuff around chain here? Because in that case it would be appropriate to update other use cases as well, for example like Kiosk you case where we're talking about Adding customer, which is exactly same problem - it has `ChainId` because it supports portfolio.

Just something to think about 🤔

Reply:
Interesting, good call. I guess `Accounts` and `Customers` are at the chain level as well? Reviewing this, I think we can't avoid mentioning `Chain`. If it needs explanation elsewhere, then we should do that as well, although the **Kiosk** use case is a little different because adding customers is more of an aside. I was already thinking about how to move the portfolio and chain explanation out of **Portfolio Access Tokens** and into **Concepts**, so this might form part of that.

Reply:
@MikeAdamsMews I think it's partially mentioned here https://mews-systems.gitbook.io/connector-api/guidelines/multi-property#how-can-i-tell-if-an-api-operation-supports-portfolio-access-tokens but only for `EnterpriseId`/`EnterpriseIds`

Reply:
Interesting. Another good call 👍 That piece of content is still valid, however I agree it would be good to update it to mention `ChainId` as well as `EnterpriseId`. **Get all loyalty programs** is an example of an API Operation that works at the multi-chain level not the multi-enterprise level, just like **Update accounts** and **Add customer**. I had not appreciated this before. I suggest not making any changes within this PR, but I will review how we talk about chains and portfolios from the ground up and make changes in a separate PR. Nothing in the current docs is invalid, I just think we could explain it better. Thanks for making me aware 👍